components/brave_rewards/browser/database/database_promotion.cc

```diff
@@ -12,6 +12,11 @@
 #include "sql/statement.h"
 #include "sql/transaction.h"
 
+namespace {
+  const char* table_name_ = "promotion";
```

Comment:
nit: `const char[] table_name = "promotion"`

Reply:
not a member variable so does not require the underscore

Reply:
covered in https://github.com/brave/brave-browser/issues/7145. This code was not added it was just moved from h file